### DISCUSSION TITLE
Gerbera and CDS:Search

### DISCUSSION THREAD

**Discussion by sunjaxx** [gerbera/gerbera#1456] [Q&A]
Hello,

I am new to Debian, so please be gentle :-)

I just installed Gerbera ("sudo apt install gerbera"). I can see / access the web front end, and can see that songs are correctly scanned. But when I try to access the library from a windows computer (with the Jriver player), I can see the DLNA server, but no songs/files are detected. I read in a Jriver forum:

_"the UPNP Media Server Content Directory Service comprises two functions — namely CDS:Browse, and CDS:Search. MC requires and uses CDS:Search(*) to download the complete library. We suppose that your server doesn’t support CDS:Search(*)"_

One final comment mentioned:

_"I updated the Gerbera server software by downloading the latest resources and now it seems to work. "_

Can you confirm if the search function is available in (the latest?) Gerbera? And if so how can I update to the latest version (I understand that in Debian I don't get the latest one...?). I also tried docker but couldn't get it to work...

Thanks!

**Comment by sunjaxx** [comment]
Update: I managed to install the docker version. This seems to work with the Jriver controller.

However, gerbera stops after few minutes, with this error:

gerbera    | 2021-05-20 19:48:34  error: Fatal error when sending updates: [/gerbera_build/src/database/sqlite3/sqlite_database.cc:356] exec Error: [/gerbera_build/src/database/sqlite3/sqlite_database.cc:435] addTask Error: sqlite3 task queue is already closed
gerbera    | 2021-05-20 19:48:34  error: Forcing Gerbera shutdown.
gerbera    | 2021-05-20 19:48:34  error: prematurely shutting down.

Any ideas?
Thanks!

**Reply by KarlStraussberger** [reply]
That's a known issue. You need to disable transactions https://docs.gerbera.io/en/stable/config-server.html#storage

Should be fixed in the next release.
